From 2ddda577681070a34a189714437e961858cd6387 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Wed, 19 Aug 2020 10:28:46 +0100
Subject: [PATCH] Remove support for the obsolete gnome-mime-data system

GNOME has used the freedesktop.org shared MIME infrastructure since
the release of GNOME 3 in 2011, and possibly longer. gnome-mime-data
was used by gnome-vfs, which was finally removed from Debian in 2019.

With gnome-mime-data disabled, debian/patches/652153_drop_gmd.patch has
no practical effect and can be dropped.

Closes: #947900
---
 debian/control                       |  3 +--
 debian/patches/652153_drop_gmd.patch | 12 ------------
 debian/patches/series                |  1 -
 debian/rules                         |  1 +
 4 files changed, 2 insertions(+), 15 deletions(-)
 delete mode 100644 debian/patches/652153_drop_gmd.patch

diff --git a/debian/control b/debian/control
index adf0e43..1beb778 100644
--- a/debian/control
+++ b/debian/control
@@ -5,8 +5,7 @@ Maintainer: Debichem Team <debichem-devel@lists.alioth.debian.org>
 Uploaders: Michael Banck <mbanck@debian.org>,
            Daniel Leidert <dleidert@debian.org>
 Build-Depends: debhelper (>> 11~)
-Build-Depends-Indep: gnome-mime-data,
-                     intltool,
+Build-Depends-Indep: intltool,
                      librsvg2-bin,
                      pkg-config,
                      shared-mime-info,
diff --git a/debian/patches/652153_drop_gmd.patch b/debian/patches/652153_drop_gmd.patch
deleted file mode 100644
index 8848ee4..0000000
--- a/debian/patches/652153_drop_gmd.patch
+++ /dev/null
@@ -1,12 +0,0 @@
-Author: Daniel Leidert <dleidert@debian.org>
-Description: gnome-mime-data-2.0 is dropped from the pkg-config file
-Bug-Debian: http://bugs.debian.org/652153
-
---- a/chemical-mime-data.pc.in
-+++ b/chemical-mime-data.pc.in
-@@ -19,4 +19,4 @@
- Description: Chemical MIME and file type support for KDE, GNOME and ROX desktops.
- Version: @VERSION@
- URL: http://chemical-mime.sf.net
--Requires:@CHEMICAL_MIME_DEPS@
-+Requires: shared-mime-info
diff --git a/debian/patches/series b/debian/patches/series
index 9ef1f2b..fe24d19 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -1,3 +1,2 @@
 sourceforge.patch
 661326_rsvg-convert.patch
-652153_drop_gmd.patch
diff --git a/debian/rules b/debian/rules
index 2595ab9..6c3fc28 100755
--- a/debian/rules
+++ b/debian/rules
@@ -14,6 +14,7 @@ override_dh_autoreconf:
 override_dh_auto_configure:
 	dh_auto_configure -- \
 	    --disable-nls \
+	    --without-gnome-mime \
 	    --without-kde-mime \
 	    --disable-update-database --disable-convert \
 	    RSVG="/usr/bin/rsvg-convert" \
-- 
2.28.0

